Sources/Testing/ABI/Encoded/ABI.EncodedIssue.swift

```diff
@@ -30,8 +30,17 @@ extension ABI {
     var _severity: Severity
 
     /// Whether or not this issue is known to occur.
+    ///
+    /// This should perhaps be deprecated in favor of `_knownIssueContext`.
     var isKnown: Bool
 
+    /// An ``Issue/KnownIssueContext-swift.struct`` representing the
+    /// ``withKnownIssue(_:isIntermittent:sourceLocation:_:when:matching:)`` call
+    /// that matched this issue, if any.
+    ///
+    /// - Warning: Known issues are not yet part of the JSON schema.
```

Comment:
Well this is just untrue.

Reply:
Hm. I copied this comment from `var _severity` just above. I took it to mean that these properties are not part of the documented JSON *schema*, even though they are part of the JSON *output*. I'd be happy to remove this here (and from `_severity` if you'd like) if you think this is unhelpful.

Reply:
The concept of issue severity is not yet officially supported and is experimental only; it needs to be made a feature through the Swift Evolution process. Known issues are already API and _are_ part of the supported schema (via the `isKnown` field). The specific changes you've made, on the other hand, are what's not yet supported.

Reply:
I removed this property.